tutorials/plot_stats_cluster_1samp_test_time_frequency.py

```diff
@@ -72,17 +73,20 @@
 # evoked_data = np.mean(data, 0)
 
 # Factor to down-sample the temporal dimension of the PSD computed by
-# single_trial_power.  Decimation occurs after frequency decomposition and can
+# tfr_transform.  Decimation occurs after frequency decomposition and can
 # be used to reduce memory usage (and possibly computational time of downstream
 # operations such as nonparametric statistics) if you don't need high
 # spectrotemporal resolution.
 decim = 5
 frequencies = np.arange(8, 40, 2)  # define frequencies of interest
 sfreq = raw.info['sfreq']  # sampling in Hz
-epochs_power = single_trial_power(data, sfreq=sfreq, frequencies=frequencies,
-                                  n_cycles=4, n_jobs=1,
-                                  baseline=(-100, 0), times=times,
-                                  baseline_mode='ratio', decim=decim)
+epochs_power = tfr_transform(data, frequencies, sfreq=sfreq, n_cycles=4.,
```

Comment:
I think it is nice to have one public interface for time-frequency, similar to `ft_freqanalysis` in fieldtrip. Also - tfr_transform seems to me to be too high-level to be private.